docs/ExpectAPI.md

```diff
@@ -646,6 +647,45 @@ test('the house has my desired features', () => {
 });
 ```
 
+### `toHaveProperty(propPath, value)`
```

Comment:
I can add `key` if that works. However, this can be a dot notation reference, in that case it's not a key name, but a deep reference. How about `refKey`?